src/data/tribers/sarika.md

```diff
@@ -0,0 +1,11 @@
+---
+name: 'Sarika'
+words:
+  - Front-end web development
+  - Learning
+  - Animals
+image: images/sarika.jpg
+story: stories/sarika.md
+---
+
+I completed my bachelor degree in computer science from India and then started a job as a SEO executive for few years and then I got interested in front-end development. I worked as a front end developer for 1.5 years and then took a long break (Maternity). Now i want start once again with my carrer in front-end development but i was so confuse where to start with. Then Neontribe gave me a chance to start again. At Neontribe i will work as a 'Frontend dev intern'.
```

Comment:
2 different spellings of front end front-end. Pick either and make them the same.

Reply:
Uppercase the letter `i` even when it is middle of a sentence.

Reply:
Change carrer to career

Reply:
OMG I did so silly mistakes …. thank you for correcting me katja